docs/contributing.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+
+## I'm non-technical, how do I contribute?
+
+If you're non-technical, there are still many ways to contribute.
+
+* You can give us **feedback** on this documentation, using [github](https://github.com/OpenVoiceOS/community-docs)
+* You can join our community through our [Chat](https://matrix.to/#/!XFpdtmgyCoPDxOMPpH:matrix.org?via=matrix.org)
+* You can contribute your **voice** to the [Community Wake Word Dataset](https://github.com/OpenVoiceOS/ovos-ww-community-dataset)
+
+## I'm technically minded, how do I contribute?
+
+Great! Let's get you started.
+
+### Reporting Issues
+
+* You'll need a [GitHub account](https://github.com/signup/free)
+* Check our [Issues](https://github.com/issues?user=OpenVoiceOS) first to see if the Issue has already been reported.
+
+If not, you'll need to create a new Issue. Help us to help fix the Issue quicker by following these guidelines. Your Issue should contain:
+
+#### A description of the Issue in object-deviation format
+
+Object-deviation format is a very specific way of describing faults. The more specific you can be, the easier it is for us to fix the fault. Begin with the _object_, such as function, a skill, or a hardware feature, then explain the _deviation from the expected condition_.
+
+For example:
+
+* "When my Mycroft Mark 1 device \(**object**\) powers on, the color of his eyes is red instead of blue \(**deviation - red - from expected condition - blue**\)
+* "When I ask about the weather \(**object**\), Mycroft reports the low temperature correctly, but the high temperature is incorrectly reported as the low temperature \(**deviation - incorrect temperature - from expected condition - correct temperature**
+
+### Making changes
+
+1. [Fork the Project](https://help.github.com/articles/fork-a-repo/)
+2. [Create a new Issue](https://help.github.com/articles/creating-an-issue/) if one doesn't already exist.
+3. Create a **feature** or **bugfix** branch based on **dev** with your issue identifier. For example, if your issue identifier is: **issue-123** then you will create either: **feature/issue-123** or **bugfix/issue-123**. Use **feature** prefix for issues related to new functionalities or enhancements and **bugfix** in case of bugs found on the **dev** branch
+4. Make sure you stick to the coding style and OO patterns that are used already.
+5. Document code using [Google-style docstrings](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).  All functions and class methods that are expected to be called externally should include a docstring.  \(And those that aren't [should be prefixed with a single underscore](https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references)\).
+6. Make commits in logical units and describe them properly. Use your issue identifier at the very begin of each commit. For instance:
```

Comment:
Not sure we need to be so specific, most of us don't follow this... A better example IMO:


Make commits in logical units with useful descriptions. For example, `troubleshooting` is not a useful commit message, but `revert module name change to troubleshoot test failures` is better